tests/instances.tcl

```diff
@@ -405,7 +410,16 @@ proc run_tests {} {
 
 # Print a message and exists with 0 / 1 according to zero or more failures.
 proc end_tests {} {
-    if {$::failed == 0} {
+    set sentinel_fd_leaks_file "sentinel_fd_leaks"
+    if { [file exists $sentinel_fd_leaks_file] } {
+        set current_dir [pwd]
+        puts [colorstr red "WARNING: sentinel test(s) failed, there are leaked fds in sentinel:"] 
+        exec cat $sentinel_fd_leaks_file
+        puts [colorstr red "see output file: $current_dir/$sentinel_fd_leaks_file for more details."]
```

Comment:
i suppose we no longer need to print the file name if we print it's content

Reply:
This output file could be for users to persist the fd leaks result in their local environment in case they miss the std output?

Reply:
but then they'll miss the WARNING message above, and this message which states the path to the file anyway.

Reply:
I was talking about the case that it's not their first run for this test suite and already know where this file will locate.

Reply:
considering we don't expect this test to ever fail, unless some regression happens, no one will know about the existence of that file, until the CI will fail the day after the offensive PR is merged.
and then we'll get the full stdout in the output.

Reply:
So we just comment out these code lines in the present?

Reply:
maybe we have a misunderstanding.. for some reason github shows as if i commented on the entire block.
i mean to just remove the last line (the one that prints the file name).
the WARNING and `cat` should be kept, and i think that's sufficient.

Reply:
I meant to only comment out the lines that prints the file path and switch it on when we need it.

Reply:
See the latest commit.

Reply:
Never mind, it doesn't indeed seem to have much value, removed it.